tests/cypress/e2e/spec.cy.ts

```diff
@@ -71,6 +71,13 @@ describe('e2e tests', () => {
 
     cy.assertHoverSelectorsOff(4);
     cy.assertHoverSelectorsOn(4);
+
+    cy.get('[class="css-1a8393j-button"]').eq(3).click(); // Remove filter by hostname
```

Comment:
That selector/eq combination seems a bit fishy to me. Can you invest a bit of time in finding a selector only solution?

If there is none, I could live with that as well.

Reply:
Please give a look at the latest commit. I introduced the query-editor selector to be more specific about the 'button' element. Moreover, I increased the strength of the assertion(s) over the desired text.